sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -967,6 +967,28 @@ object SQLConf {
     .booleanConf
     .createWithDefault(false)
 
+  val FILE_META_CACHE_ORC_ENABLED = buildConf("spark.sql.fileMetaCache.orc.enabled")
+    .doc("To indicate if enable orc file meta cache, it is recommended to enabled " +
+      "this config when multiple queries are performed on the same dataset, default is false.")
+    .version("3.3.0")
+    .booleanConf
+    .createWithDefault(false)
```

Comment:
Should we take the approach to have a list config for all data sources enabled with file meta cache, instead of one config per data source?

Reply:
+1 for @viirya 's suggestion.

Reply:
ok

Reply:
ebb7e0b rename config name